pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/PillarboxPlayer.kt

```diff
@@ -126,6 +127,18 @@ class PillarboxPlayer internal constructor(
     }
 }
 
+/**
+ * Get a snapshot of the current media items
+ */
+fun Player.getCurrentMediaItems(): Array<MediaItem> {
```

Comment:
Maybe return a `List` instead?

Reply:
Yes good point as the list have to be immutable.